torch/ao/quantization/fx/quantization_patterns.py

```diff
@@ -34,18 +43,24 @@
 class QuantizeHandler(ABC):
     """ Base handler class for the quantizer patterns
     """
-    def __init__(self, node: Node, modules: Dict[str, torch.nn.Module]):
+    def __init__(
+            self,
+            node_pattern: NodePattern,
+            modules: Dict[str, torch.nn.Module],
+            root_node_getter: Callable = None):
```

Comment:
what's the context to needing to override this? can this be avoided?

Reply:
this is not need for current fbgemm patterns, but needed to support more complex patterns, e.g. conv - add - relu

Reply:
e.g. 
(nn.ReLU, (operator.add, (nn.BatchNorm2d, nn.Conv2d), MatchAllNode))

Reply:
would it make sense to add this in the future when it's needed?  I wasn't sure how this is used in this PR

Reply:
the code path is exercised in the next PR, but I'll add a test later to test the custom root_node_getter functions, this is currently not a priority and we would like to finish the frontend refactor asap